src/org/opendatakit/briefcase/ui/reused/transfer/sourcetarget/source/CustomDir.java

```diff
@@ -103,7 +103,6 @@ public void storeSourcePrefs(BriefcasePreferences prefs, boolean storePasswords)
   public JobsRunner pull(TransferForms forms, BriefcasePreferences appPreferences, BriefcasePreferences localPreferences) {
     return JobsRunner.launchAsync(forms.map(form -> run(jobStatus -> transferODKToBriefcase(
         appPreferences.getBriefcaseDir().orElseThrow(BriefcaseException::new),
-        path.toFile(),
```

Comment:
old/legacy

This comes from the change in `TransferFromODK` which should be eventually replaced by a new implementation